src/App.tsx

```diff
@@ -15,36 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  LENGTH = 'L',
+  ALPHABET = 'A',
+  DEFAULT = '',
```

Comment:
```suggestion
  Length = 'length',
  Alphabet = 'alphabet',
  DDefault = '',
```
`L`, `A`? Also, they should be in the upper case, check examples [here](https://mate-academy.github.io/fe-program/js/extra/typescript)
![image](https://github.com/user-attachments/assets/da4585b6-4366-405f-872b-e30bf89b2da3)